src/PackageManagement/AzureAutomationTools.PackageManagement.psm1

```diff
@@ -741,7 +891,11 @@ function DeployModules {
     param(
         [Parameter(Mandatory = $true)]
         [ValidateNotNullOrEmpty()]
-        [string]$Path
+        [string]$Path,
+
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
+        [int]$MaxModulePoll = 100
```

Comment:
This should be clear that this is a max number of attempts e.g. **MaxPollAttempts**.  The Poll interval should also be an optional parameter e.g. **PollIntervalSeconds**